test/cmd/templates.sh

```diff
@@ -8,7 +8,6 @@ trap os::test::junit::reconcile_output EXIT
   oc delete all,templates --all
   oc delete template/ruby-helloworld-sample -n openshift
   oc delete project test-template-project
-  wait_for_command '! oc get project test-template-project'
```

Comment:
We don't seem to need to do this on any of the other deletion steps, so I'm hesitant to keep it in.